cms/tests/test_i18n.py

```diff
@@ -364,9 +364,10 @@ def test_session_language(self):
         session = self.client.session
         session[LANGUAGE_SESSION_KEY] = 'fr'
         session.save()
-        response = self.client.get('/')
-        self.assertEqual(response.status_code, 302)
-        self.assertRedirects(response, '/fr/')
+        if not DJANGO_3_0:
```

Comment:
@yakky do you know any specific changes in Django 3 for language fullbacks, I cannot figure out what is going on there exactly.

Reply:
This is due to language not being stored in session anymore (see https://docs.djangoproject.com/en/3.0/releases/3.0/#id3 item 3)

I cannot find a better announcement regarding the change

I kept the logic unchanged on 2.2 and below to avoid changing existing behavior